cloudsmith_cli/cli/config.py

```diff
@@ -115,7 +115,7 @@ def create_default_file(cls, data=None, mode=None):
         for k, v in six.iteritems(data):
             v = v or ""
             config = re.sub(
-                r"^(%(key)s) =[ ]*$" % {"key": k},
+                r"^(%(key)s)\s?=[ ]*$" % {"key": k},
```

Comment:
Ha! Nice. Maybe it should be even more tolerant then?

`r"^(%(key)s)\s*=\s*$" % {"key": k},`